documentation/book/assembly-kafka-connect.adoc

```diff
@@ -9,15 +9,18 @@
 [id='kafka-connect-{context}']
 = Kafka Connect
 
-The Cluster Operator deploys a link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] cluster, which can be used with your Kafka broker deployment.
-It is implemented as a `Deployment` with a configurable number of workers.
-The default image currently contains only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-The REST interface for managing the Kafka Connect cluster is exposed internally within the {ProductPlatformName} cluster as a `kafka-connect` service on port `8083`.
-
-Example `KafkaConnect` resources and the details about the `KafkaConnect` format for deploying Kafka Connect can be found in
-xref:assembly-deployment-configuration-kafka-connect-str[Kafka Connect cluster configuration]
-and
-xref:assembly-deployment-configuration-kafka-connect-s2i-str[Kafka Connect cluster with Source2Image support].
+link:https://kafka.apache.org/documentation/#connect[Kafka Connect^], a core component of Apache Kafka, is a tool for streaming high volumes of data into and out of your Kafka cluster. It provides a framework for building connectors (types of plug-ins) for your Kafka cluster, a REST API for managing connectors, and more. It includes two built-in connectors:
```

Comment:
_streaming high volume_ sounds strange. Wouldn't just _transfering data into and out of ..._ be better?

Reply:
I've reconciled this with the description we have in the RHEL documentation. "It provides a framework for moving large amounts of data into and out of your cluster while maintaining scalability and reliability."

Reply:
Sounds good. Thanks.